integration_tests/test_statistics.py

```diff
@@ -97,6 +97,18 @@ def test_stdev():
     k = stdev(b)
     assert abs(k - 0.41967844833872525) < eps
 
+def test_linear_regression():
+    c: list[f64]
+    c = [2.74, 1.23, 2.63, 2.22, 3.0, 1.98]
+    d: list[f64]
+    d = [9.4, 1.23, 2.63, 22.4, 1.9, 13.98]
+
+    slope: f64
+    intercept: f64
+    slope, intercept = linear_regression(c,d)
```

Comment:
```suggestion
    slope, intercept = linear_regression(c, d)
```

Reply:
@Madhav2310

Reply:
Thank you! I'm implemented this change too, please review and merge

Reply:
> Also add the tests for linear_regression(list[i32], list[i32])